### ISSUE TITLE
Unable to get docker install to recognise that I'm outside of local network.

### ISSUE THREAD

**Issue by smarthome-enthusiast** [causefx/Organizr#1967]
###### Operating System: Synology 7.2 + Official Docker Image
<hr>

##### Problem Description:
**TL;DR: I am unable to get Official Docker image to see that I am outside of local network and use Tab URL instead of local tab URL on Synology/DSM 7.2**

I've set the local tab ip for internal 192.168.x.x and tab url to an accessible external address however when using Organizr from an external IP (100.x.x.x), none of the tabs work as it is using the Local Tab URLs to connect.

I have:
1. set the Override Local IP or Subnet to match 192.168.x.x CIDR in Organizr Login Settings. 
2. checked the debug area and it shows that Local: true even though I am not on that subnet.
3. checked the logs and it shows all activities are from 172.x.x.x .
4. set the correct "Real Docker IP" range in default site config.
5. tried the following fix from Pedro Lamas but it didn't work.
    Link: https://www.pedrolamas.com/2020/11/04/exposing-the-client-ips-to-docker-containers-on-synology-nas/
6. set Synology's Trusted Proxies setting to both Docker IP and Local IP of NAS
7. allowed traffic to subnet on Firewall.
8. used the test.php in issue #1836 and can not find the PHP Variables:
 
![image](https://user-images.githubusercontent.com/16184466/169606015-f4d09118-1ed8-4f86-ac7f-fed49a3c7548.png)

_Originally posted by @causefx in https://github.com/causefx/Organizr/issues/1836#issuecomment-1133335598_

Is there something I am missing or haven't enabled? Is the image already set up to forward IPs?

_I am somewhat familiar with Docker and CLI, but not an expert. I am a **complete** novice at web servers so any help with instructions would be appreciated_

**Comment by HalianElf** [comment]
That link you linked should be what you need. I would try going through that again since that's the only way you're going to get it to see the correct IP (and there's no way for Organizr to detect it when it's getting the wrong info).

**Comment by smarthome-enthusiast** [comment]
> That link you linked should be what you need. I would try going through that again since that's the only way you're going to get it to see the correct IP (and there's no way for Organizr to detect it when it's getting the wrong info).

I took another look at the link (If I understood correctly, you are referring to no. 8?), and it seems like these are the only PHP HTTP Variables that I'm getting:

![image](https://github.com/causefx/Organizr/assets/143326320/aae21bec-b0dc-4df6-9685-edc2cbb347f0)

I can't seem to locate the HTTP_X_REAL_IP or HTTP_X_FORWARDED_FOR variables.

**Comment by HalianElf** [comment]
No this one with the iptables stuff: https://www.pedrolamas.com/2020/11/04/exposing-the-client-ips-to-docker-containers-on-synology-nas/

It's getting the wrong IP because syno isn't properly NATing it so it's showing the from as the docker gateway

**Comment by smarthome-enthusiast** [comment]
> No this one with the iptables stuff: https://www.pedrolamas.com/2020/11/04/exposing-the-client-ips-to-docker-containers-on-synology-nas/
> 
> It's getting the wrong IP because syno isn't properly NATing it so it's showing the from as the docker gateway

Thanks for getting back to me so quickly! I went ahead and ran the script again, and now I can find the Variables:

![image](https://github.com/causefx/Organizr/assets/143326320/77b4cf47-4b2a-481b-821e-7a501b1cf88e)

But it is still showing that I am Local:

![image](https://github.com/causefx/Organizr/assets/143326320/c373ba32-4609-499e-b2f2-e8e69a87b5e9)

**Comment by HalianElf** [comment]
That definitely looks better. Are you setting an override for local IPs? 100.x.x.x should be considered remote

**Comment by smarthome-enthusiast** [comment]
Yes I've set the "Override Local IP or Subnet" to 192.x.x.x and 100.x.x.x **_should_** be considered as remote, but it still isn't. I've also noticed that it's still reporting my IP as 172.x.x.x, is this something I should be concerned about?

![image](https://github.com/causefx/Organizr/assets/143326320/54578649-eb80-438a-8376-78bc18e1884c)

![image](https://github.com/causefx/Organizr/assets/143326320/4b0ec624-4971-4c58-b395-b8de77f64527)

![image](https://github.com/causefx/Organizr/assets/143326320/71d69d65-598d-4bd8-8211-90992a018d80)

![image](https://github.com/causefx/Organizr/assets/143326320/66e5d5e0-c74a-48f8-9c80-adefe4f8b6cf)

Edit: added screenshots and made original post clearer

**Comment by smarthome-enthusiast** [comment]
> That definitely looks better. Are you setting an override for local IPs? 100.x.x.x should be considered remote

Am I supposed to fill in the WAN domain and Enable Local Address Forward when setting an override for local IPs?

**Comment by HalianElf** [comment]
You shouldn't be changing the override at all, it all ready contains all the RFC-1918 space and no you don't need do that, that's just for switching urls for Organizr when you're local.
